rollbar-java/src/main/java/com/rollbar/notifier/Rollbar.java

```diff
@@ -671,7 +697,8 @@ private Data buildData(Config config, Throwable error, Map<String, Object> custo
         .platform(config.platform())
         .language(config.language())
         .framework(config.framework())
-        .level(level != null ? level : level(error))
+        .level(level != null ? level : error != null ? level(error.getThrowable())
```

Comment:
It is possible that `getThrowable` returns `null`, but that is okay here because `level(null)` gives the default.

Reply:
Yes, if it's a represented throwable, https://logging.apache.org/log4j/2.0/log4j-core/apidocs/org/apache/logging/log4j/core/impl/ThrowableProxy.html#getThrowable(). That was why i decided to introduce the `ThrowableWrapper` to represent this kind of errors where there is no access to the class of the Exception.